### PR TITLE
German translation added

### PR DESCRIPTION
**Pull request by GitRon**
just po.file, mo is ignored by .gitignore

### CODE REVIEW

File: safedelete/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+"Language: fr\n"
+
+#: admin.py:31
+msgid "Name"
+msgstr "Name"
+
+#: admin.py:109 admin.py:116
+#, python-format
+msgid "Successfully undeleted %(count)d %(items)s."
+msgstr "%(count)d %(items)s gelöscht."
```
Comment (AndreasBackx) [inline safedelete/locale/de/LC_MESSAGES/django.po]: The English one says "undeleted" while the German one means "deleted".

File: safedelete/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+#: admin.py:129
+msgid "Are you sure?"
+msgstr "Sind Sie sicher?"
+
+#: admin.py:146
+#, python-format
+msgid "Undelete selected %(verbose_name_plural)s."
+msgstr "Ausgewählte %(verbose_name_plural)s wiederherstellen."
```
Comment (AndreasBackx) [inline safedelete/locale/de/LC_MESSAGES/django.po]: You didn't make the "undelete" mistake from L27 here.

File: safedelete/locale/de/LC_MESSAGES/django.po (hunk 1 above)
Comment (GitRon) [reply safedelete/locale/de/LC_MESSAGES/django.po]: Oops.

File: safedelete/locale/de/LC_MESSAGES/django.po (hunk 2 above)
Comment (GitRon) [reply safedelete/locale/de/LC_MESSAGES/django.po]: Yep :)

**Comment by AndreasBackx** [comment]
Looks good now! Squash and merge @Gagaro?

**Comment by Gagaro** [comment]
I usually prefer using the `--no-location` option to avoid merge hell, but this po file is pretty simple so that's all right.

Thank you @GitRon for the PR and thank you @AndreasBackx for the review :+1: .

**Pull request merged**